charts/opentelemetry-demo/values.yaml

```diff
@@ -356,13 +357,13 @@ opentelemetry-collector:
   mode: deployment
   config:
     exporters:
-      jaeger:
-        endpoint: '{{ .Release.Name }}-jaeger:14250'
+      otlp:
+        endpoint: '{{ .Release.Name }}-jaeger:4317'
```

Comment:
Interesting, I wasn't aware jaeger new how to receive otlp data

Reply:
They are also deprecating their SDKs, pushing people towards OTLP 😄